Description: skips LICENSE.txt during installation. All information is in debian/copyright.
Author: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-11-28

Index: gmsh-2.8.5-source/CMakeLists.txt
===================================================================
--- gmsh-2.8.5-source.orig/CMakeLists.txt
+++ gmsh-2.8.5-source/CMakeLists.txt
@@ -1411,7 +1411,7 @@ if(UNIX)
 endif(UNIX)
 
 set(WELCOME_FILE ${CMAKE_CURRENT_SOURCE_DIR}/doc/WELCOME.txt)
-set(LICENSE_FILE ${CMAKE_CURRENT_SOURCE_DIR}/doc/LICENSE.txt)
+#set(LICENSE_FILE ${CMAKE_CURRENT_SOURCE_DIR}/doc/LICENSE.txt)
 set(CREDITS_FILE ${CMAKE_CURRENT_SOURCE_DIR}/doc/CREDITS.txt)
 set(ONELAB_PY ${CMAKE_CURRENT_SOURCE_DIR}/contrib/onelab/python/onelab.py)
 file(GLOB TUTORIAL_FILES ${CMAKE_CURRENT_SOURCE_DIR}/tutorial/?*.*)
@@ -1576,7 +1576,7 @@ execute_process(COMMAND ${CMAKE_COMMAND}
 if(MAKEINFO AND TEXI2PDF)
   add_custom_target(doc COMMAND ${CMAKE_COMMAND} -E tar zcf 
                     ${CMAKE_CURRENT_BINARY_DIR}/gmsh-${GMSH_VERSION}-doc.tgz
-                    doc/CREDITS.txt doc/LICENSE.txt doc/VERSIONS.txt
+                    doc/CREDITS.txt doc/VERSIONS.txt
                     doc/gmsh.1 doc/texinfo/gmsh.html doc/texinfo/gmsh.info 
                     doc/texinfo/gmsh.pdf doc/texinfo/gmsh.txt
                     COMMAND ${CMAKE_COMMAND} -E remove ${TEX_OBJ}
